tests/inference/inference_utils.py

```diff
@@ -82,7 +83,10 @@ def setUpClass(cls):
 
         # Upload to the hub
         cls.neuron_model_id = f"{cls.USER}/{cls.NEURON_MODEL_REPO}"
-        neuron_model.push_to_hub(model_dir, repository_id=cls.neuron_model_id, use_auth_token=cls._token)
+
+        use_auth_token = os.environ.get("HF_TOKEN", None)
```

Comment:
Why don't you check cls._token instead ?

Reply:
Good catch! Will use directly cls._token instead.